components.md

```diff
@@ -0,0 +1,98 @@
+# Components
+
+This page provides details on the main component/entity types that make up a deployment of the ECP.
+
+## Director
+A Director manages a part of or the entire setup, orchestrating the actors and observers according to the needs of the experiment.
+It can, among other things, 
+* issue commands to actors
+* request data from individual actors
+* read data from an observer
+
+Potentially a GUI could be attached here too.
+
+## Action
+An Action represents a method or function of the device represented by a Driver.
+It can be called with zero or more arguments.
+
+## Actor
+The Actor is a component which (typically) instantiates one or more Driver class and/or does some kind of processing.
+Actions which the actor can take could be anything from measuring on a regular basis ("acting" as a sensor), commanding hardware devices through the use of the driver to do X or Y, processing some data and re-emitting results (e.g. a PID controller).
+The actor is part of the communication network, that means it is listening for messages and maybe sending messages by itself.
```

Comment:
I would put that part on top of the description, as I understand it as the main job:

The actor participates in the network listening for commands and acts on them. It may act regularly at its own account.
Typically an actor controls one or several devices via Drivers...

Reply:
Makes sense!